index.d.ts

```diff
@@ -1,9 +1,8 @@
-import {Buffer} from 'node:buffer';
-import {Readable as ReadableStream} from 'node:stream';
+import type {Readable as ReadableStream} from 'node:stream';
 
 declare const isProgressive: {
 	/**
-	Checks if a `Buffer` contains a JPEG image that is [progressive](http://www.faqs.org/faqs/jpeg-faq/part1/section-11.html).
+	Checks if an `ArrayBuffer/UInt8Array` contains a JPEG image that is [progressive](http://www.faqs.org/faqs/jpeg-faq/part1/section-11.html).
```

Comment:
The type only indicates it supports `UInt8Array`.

Reply:
Changed it to `Uint8Array`, plus I keep capitalizing `UInt` instead of `Uint`, bad habit.